src/main/java/uk/gov/justice/hmpps/prison/service/AppointmentsService.java

```diff
@@ -100,8 +97,17 @@ public void createAppointments(@NotNull @Valid final AppointmentsToCreate appoin
 
         final var withRepeats = withRepeats(appointments.getRepeat(), flattenedDetails);
```

Comment:
I will refactor this if we go with this solution.. but right now we just need to test it.